users/views.py

```diff
@@ -1,3 +1,50 @@
-from django.shortcuts import render
+import json, requests, jwt
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+
+from users.models import User
+
+from my_settings import SECRET_KEY, ALGORITHM
+
+class KakaoSignInView(View):
+    def get(self, request):
+        try:
+            kakao_access_token = request.headers.get('Authorization')
+
+            headers = {
+                'Authorization': f'Bearer {kakao_access_token}'
+            }
+
+            kakao_user_info = requests.post('https://kapi.kakao.com/v2/user/me', headers=headers).json()
+
+            kakao_id  = kakao_user_info['id']
+            email     = kakao_user_info['kakao_account']['email']
+            nick_name = kakao_user_info['properties']['nickname']
+            birthday  = kakao_user_info['kakao_account']['birthday']
+            gender    = kakao_user_info['kakao_account']['gender']
+
+            user, is_created =  User.objects.get_or_create(
+                kakao_id  = kakao_id,
+                email     = email,
+                nick_name = nick_name,
+                birthday  = birthday,
+                gender    = gender
+            )
```

Comment:
get_or_create을 사용할 경우 해당 데이터를 특정 지울 수 있는 것으로 먼저 체크 후 생성 되었을 때 데이터를 넣어 주어야 합니다.
```suggestion
            user, is_created =  User.objects.get_or_create(kakao_id  = kakao_id)
            
            if is_created:
                user.email     = email,
                user.nick_name = nick_name,
                user.birthday  = birthday,
                user.gender    = gender
                user.save()
``` 
또는 default 옵션을 사용할 수 있는데, 이 부분은 공식 문서를 참고해 보세요.

Reply:
넵 반영했습니다!! 문서도 참고해보겠습니다